### DISCUSSION TITLE
Update redlib to use current icon instead of libreddit's

### DISCUSSION THREAD

**Discussion by Elbullazul** [selfhst/icons#144] [Icon Update Request]
I don't know when the logo changed, but the project now uses a flipped and recolor version:

![new-logo](https://raw.githubusercontent.com/redlib-org/redlib/refs/heads/main/static/logo.png)

[PNG version](https://github.com/redlib-org/redlib/blob/main/static/logo.png) [SVG version](https://github.com/redlib-org/redlib/blob/main/static/logo.svg)

**Comment by shollyethan** [comment]
Good catch! I've updated the logo in my collection and manually triggered a refresh of jsDelivr's cache. It may take a bit for the updated icon to appear.
